FIX: Don't force recipe usage

lozcalver opened the pull request:
We don’t use the recipes directly, so when this module gets installed (and the recipe along with it) a bunch of recipe files like `mysite.yml` get created alongside it.

I’ve only included framework and siteconfig in the base requirements: as far as I can tell these are the only two _required_ for this module to run. CMS/assets modules are referenced in places, but as things like `instanceof ` checks, so shouldn’t be  directly necessary outside of tests.

lozcalver committed:
FIX: Don't force recipe usage

tractorcow commented:
Agreed with this, I probably should have not been so restrictive on it from the start.

lozcalver commented:
Thanks @tractorcow! Could I bug you tag a release too? Or if you don’t mind inviting me to the repo I can do it

Looks like the two merged PRs are the only changes since the last tag: https://github.com/tractorcow/silverstripe-opengraph/compare/5.1.0...master

lozcalver commented:
Ping @tractorcow 👼
